image_generation/common/diffusers/src/scheduler_lcm.cpp

```diff
@@ -31,10 +31,11 @@ std::vector<T> linspace(U start, U end, size_t num, bool endpoint = false) {
 }
 
 // for torch.randn()
-std::vector<float> randn_function(uint32_t size, uint32_t seed = 42) {
+std::vector<float> randn_function(uint32_t size) {
     std::vector<float> noise(size);
     {
-        std::mt19937 gen{static_cast<unsigned long>(seed)};
+        std::random_device rd;
+        std::mt19937 gen(rd());
```

Comment:
The image quality of master version without reading noise is worse than reading noise with "-r". Please check with [this link](https://github.com/openvinotoolkit/openvino.genai/tree/master/image_generation/lcm_dreamshaper_v7/cpp).

Reply:
How do you define and measure image quality? To me the images generated from master with --seed 0, 1 ,2 and 3 look fine. Anyway, your results aren't reproducible because the seed isn't fixed. I think you were just lucky to get a seed from random_device which produced the images you like more.

The link you provided is just `lcm_dreamshaper_v7/cpp` folder. What should I check there?